lib/workers/repository/update/branch/execute-post-upgrade-commands.spec.ts

```diff
@@ -520,5 +525,87 @@ describe('workers/repository/update/branch/execute-post-upgrade-commands', () =>
         },
       ]);
     });
+
+    it('uses workingDirTemplate when provided', async () => {
+      const commands = partial<BranchUpgradeConfig>([
+        {
+          manager: 'npm',
+          branchName: 'renovate/jest-29.x',
+          depName: 'jest',
+          newVersion: '29.5.0',
+          postUpgradeTasks: {
+            commands: ['some-command'],
+            workingDirTemplate:
+              '/projects/{{manager}}/{{depName}}-{{newVersion}}',
```

Comment:
enforce that it's relative below repo root, otherwise it can be executed in any folder